modal/output.py

```diff
@@ -13,6 +13,9 @@
     from ._output import OutputManager
 
 
+OUTPUT_ENABLED = False
```

Comment:
confused why this is needed vs checking the singleton?

Reply:
Right now we can't check the singleton without importing the `_output` module, which has a ton of rich imports in global scope. We could move those imports into the relevant function bodies (and leave some guarded under `if TYPE_CHECKING` for rich types used in the `OutputManager` interfaces) but that's cumbersome/hacky in its own way.

I'd prefer to turn the `OutputManager` into more of a _manager_ with no display responsibilities itself, and make the code that uses rich directly more modular and easier to replace with plain versions in contexts where rich isn't available.